contrib/lib/src/databases.rs

```diff
@@ -700,6 +707,27 @@ impl Poolable for memcache::Client {
     }
 }
 
+#[cfg(feature = "arango_pool")]
+impl Poolable for arangors::Connection {
+    type Manager = r2d2_arangors::pool::ArangoDBConnectionManager;
+    type Error = r2d2::Error;
+
+    fn pool(db_name: &str, rocket: &rocket::Rocket) -> PoolResult<Self> {
+        #[derive(Deserialize, Debug)]
+        struct ArangoConfig {
```

Comment:
Where are these configuration options documented? We'll need them somewhere, otherwise it'll be impossible for someone to properly configure this database and thus use it at all.

Reply:
That's right, i will write something about that. Is it ok to add it here, right above the `## Extended`?
https://github.com/SergioBenitez/Rocket/blob/77faddfc8b458f5b23135f86fa857573cc18d81c/contrib/lib/src/databases.rs#L352-L353

Reply:
I've added docs at that position. Feel free to move them around if that is the wrong place 😅